config/config.proto

```diff
@@ -70,6 +70,17 @@ message SlackConfig {
   optional bool send_resolved = 5 [default = false];
 }
 
+message FlowdockConfig {
+  // Flowdock flow API token
```

Comment:
Sorry if I didn't notice this before, but style nit: period after each doc comment please.